lib/include.js

```diff
@@ -437,8 +437,9 @@ Inclusion.include = function(objects, include, options, cb) {
       if (polymorphic) {
         // handle polymorphic hasMany (reverse) in which case we need to filter
         // by discriminator to filter other types
+        var throughModel = polymorphic.invert ? relation.modelTo : relation.modelFrom;
```

Comment:
let's use `const` instead of `var`, see eslint error:

```
> loopback-datasource-juggler@4.8.2 lint /home/jenkins/workspace/nb/loopback-datasource-juggler~master/37975cf6
11:22:41 > eslint .
11:22:41 
11:22:59 
11:22:59 /home/jenkins/workspace/nb/loopback-datasource-juggler~master/37975cf6/lib/include.js
11:22:59   440:9  error  Unexpected var, use let or const instead  no-var
11:22:59 
11:22:59 ✖ 1 problem (1 error, 0 warnings)
11:22:59   1 error and 0 warnings potentially fixable with the `--fix` option.
```